pkg/kubelet/kubelet.go

```diff
@@ -2693,6 +2698,31 @@ func (kl *Kubelet) doneNetworkConfigure() bool {
 	return kl.networkConfigured
 }
 
+func (kl *Kubelet) containerRuntimeVersionRequirementMet() bool {
+	kl.runtimeMutex.Lock()
```

Comment:
Yeah, we don't need the lock here now. My initial implementation is combining both containerRuntimeUp and version check together which requires a lock. We don't need this  now since they are separated.